neurom/core/morphology.py

```diff
@@ -523,37 +497,37 @@ def __hash__(self):
         """Hash is made of tuple of type and root_node."""
         return hash((self.type, self.root_node))
 
-    __bool__ = __nonzero__
-
     def __repr__(self):
         """Return a string representation."""
         return 'Neurite <type: %s>' % self.type
 
 
-class Morphology(morphio.mut.Morphology):
+class Morphology:
     """Class representing a simple morphology."""
 
     def __init__(self, filename, name=None):
         """Morphology constructor.
 
         Args:
-            filename (str|Path): a filename
-            name (str): a option morphology name
+            filename (str|Path): a filename or morphio.{,mut.}Morphology object
+            name (str): a optional morphology name
```

Comment:
```suggestion
            name (str): an optional morphology name
```
I am not sure but I would use `an` in this case.

Reply:
Fixed